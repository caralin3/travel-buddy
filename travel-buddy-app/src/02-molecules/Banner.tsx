import React from 'react';
import { Container } from 'reactstrap';

export interface BannerProps {}

export const Banner: React.FC<BannerProps> = ({ children }) => (
  <Container fluid className="px-2 py-3 bg-info border-bottom border-top">
    {children}
  </Container>
);
