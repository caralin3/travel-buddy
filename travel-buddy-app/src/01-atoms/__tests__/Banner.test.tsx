import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Banner } from '../Banner';

describe('Banner', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Banner />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
