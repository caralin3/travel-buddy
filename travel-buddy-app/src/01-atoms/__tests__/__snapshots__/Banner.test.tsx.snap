// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should render correctly 1`] = `
<Container
  className="px-2 py-3 bg-secondary text-white border-bottom border-top"
  fluid={true}
  tag="div"
/>
`;
